examples/hybrid_jobs/7_Running_notebooks_as_jobs/Running_notebooks_as_jobs.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>Why is this function needed, when <a href="https://github.com/aws/amazon-braket-sdk-python/blob/main/src/braket/aws/aws_quantum_job.py#L410" target="_blank">AwsQuantumJob.result</a> does the same thing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
thanks! updated it to download_result!